packages/framework/src/Component/Reflection/ReflectionHelper.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component\Reflection;
+
+use ReflectionClass;
+use ReflectionClassConstant;
+
+class ReflectionHelper
```

Comment:
I would like to see this as a base abstract class for enums, with method named like `slices` or `getSlices` so it feels more like an enum